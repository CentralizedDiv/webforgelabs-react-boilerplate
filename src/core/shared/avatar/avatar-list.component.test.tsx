import React from 'react';
import STAvatarList, { ISTAvatarListProps } from './avatar-list.component';
import { shallow } from 'enzyme';


describe('AvatarList', () => {
  const props: ISTAvatarListProps = {
    names: ['Jos√©', 'Maria', 'Ana'],
  };
  const component = shallow(<STAvatarList {...props} />);

  it('renders properly', () => {
    expect(component).toMatchSnapshot();
  });

});