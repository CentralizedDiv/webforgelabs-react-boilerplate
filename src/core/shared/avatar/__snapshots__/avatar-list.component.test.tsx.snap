// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AvatarList renders properly 1`] = `
<div>
  <Tooltip
    arrowPointAtCenter={false}
    autoAdjustOverflow={true}
    key="José"
    mouseEnterDelay={0.1}
    mouseLeaveDelay={0.1}
    placement="top"
    title="José"
    transitionName="zoom-big-fast"
  >
    <STAvatar
      key="José"
      small={true}
      style={
        Object {
          "marginRight": "4px",
        }
      }
    >
      JO
    </STAvatar>
  </Tooltip>
  <Tooltip
    arrowPointAtCenter={false}
    autoAdjustOverflow={true}
    key="Maria"
    mouseEnterDelay={0.1}
    mouseLeaveDelay={0.1}
    placement="top"
    title="Maria"
    transitionName="zoom-big-fast"
  >
    <STAvatar
      key="Maria"
      small={true}
      style={
        Object {
          "marginRight": "4px",
        }
      }
    >
      MA
    </STAvatar>
  </Tooltip>
  <Tooltip
    arrowPointAtCenter={false}
    autoAdjustOverflow={true}
    key="Ana"
    mouseEnterDelay={0.1}
    mouseLeaveDelay={0.1}
    placement="top"
    title="Ana"
    transitionName="zoom-big-fast"
  >
    <STAvatar
      key="Ana"
      small={true}
      style={
        Object {
          "marginRight": "4px",
        }
      }
    >
      AN
    </STAvatar>
  </Tooltip>
</div>
`;
